browser-test/src/applicant/northstar_common_intake_upsell.test.ts

```diff
@@ -80,6 +80,39 @@ test.describe(
       })
     })
 
+    test('As guest, validate create account button', async ({
+      page,
+      adminPrograms,
+      applicantQuestions,
+    }) => {
+      await test.step('Setup: publish one program', async () => {
+        await adminPrograms.addProgram(eligibleProgram1)
+        await adminPrograms.publishProgram(eligibleProgram1)
+        await logout(page)
+      })
+
+      await enableFeatureFlag(page, 'north_star_applicant_ui')
+
+      await test.step('Setup: submit application', async () => {
+        await applicantQuestions.clickApplyProgramButton(programName)
+        await applicantQuestions.submitFromReviewPage(
+          /* northStarEnabled= */ true,
+        )
+      })
+
+      await test.step('Validate the create account button navigates to the correct page', async () => {
+        await expect(
+          page.getByText(
+            'Create an account to save your application information',
+          ),
+        ).toBeVisible()
+        await applicantQuestions.clickCreateAnAccount()
+        await expect(page).toHaveURL('http://dev-oidc:3390/') // Auth page for browser tests
```

Comment:
(blocking) You can't do it this way. It won't work on staging websites. You need to be able to support the same options login does. Check the `browser-test/src/support/index.ts` file. Look for the `loginAs...` functions and how they operate.

If you still end up needing the dev-oidc url string add it to `browser-test/src/support/config.ts`, but ideally you won't need it if you follow the pattern for the login.

Reply:
Ok, that fixed the login link. I looked at some other tests and they don't test the create account button on the homepage either. Is it OK to not have a test for the "Create an account" button?

Reply:
Yeah. I think just checking the buttons exist is fine and not to worry about checking redirects.